server/service/individualServices/PrepareForwardingAutomation.js

```diff
@@ -50,4 +55,47 @@ exports.OAMLayerRequest = function (uuid) {
       reject(error);
     }
   });
+}
+
+/**
+ * @description This function automates the forwarding construct by calling the appropriate call back operations based on the fcPort input and output directions.
+ * @param {String} forwardingKindName forwarding Name
+ * @param {list}   attributeList list of attributes required during forwarding construct automation(to send in the request body)
+ * @param {String} user user who initiates this request
+ * @param {string} originator originator of the request
+ * @param {string} xCorrelator flow id of this request
+ * @param {string} traceIndicator trace indicator of the request
+ * @param {string} customerJourney customer journey of the request
+ * @param {Boolean} isMethodGET true if method of the request is GET
+ * @param {Object} params path and query params
+ **/
+exports.forwardRequest = async function (forwardingKindName, attributeList, user, xCorrelator, traceIndicator, customerJourney, isMethodGET, params) {
+  let forwardingConstructInstance = await ForwardingDomain.getForwardingConstructForTheForwardingNameAsync(forwardingKindName);
```

Comment:
Kindly please consider the "RequestForProvidingAcceptanceDataCausesDeterminingTheLanPortRole" scenario, since we have multiple OUTPUT fc-port.